include/nccl_ofi_param.h

```diff
@@ -179,9 +179,10 @@ OFI_NCCL_PARAM_INT(disable_native_rdma_check, "DISABLE_NATIVE_RDMA_CHECK", 0);
 OFI_NCCL_PARAM_INT(disable_gdr_required_check, "DISABLE_GDR_REQUIRED_CHECK", 0);
 
 /*
- * Maximum size of a message in bytes before message is multiplexed
- */
-OFI_NCCL_PARAM_INT(round_robin_threshold, "ROUND_ROBIN_THRESHOLD", (256 * 1024));
+ * Minimum data size in bytes before the message is to be striped across multiple 
+ * rails.
+*/
```

Comment:
Nit: This comment is a bit confusing still. Stated simply: Messages sized larger than this threshold will be striped across multiple rails.

Reply:
Done